tasty-query/shared/src/test/scala/tastyquery/BaseUnpicklingSuite.scala

```diff
@@ -8,4 +8,6 @@ import tastyquery.testutil.TestPlatform
 
 abstract class BaseUnpicklingSuite extends munit.FunSuite {
   lazy val testClasspath: Future[Classpath] = TestPlatform.loadClasspath()
+
+  override def munitTimeout = scala.concurrent.duration.Duration.Inf
```

Comment:
overlooked

Reply:
That was on purpose, in a dedicated commit, so that it remains there for all our future debugging sessions.